src/Image.php

```diff
@@ -176,15 +176,27 @@ public function setImportantPart(ImportantPartInterface $importantPart = null)
      */
     private function getSvgSize()
     {
+        static $zlibSupport;
```

Comment:
I don’t think so, because it is only used in this method and nowhere else. 
The exception would be if our styleguide doesn’t allow static function variables at all.